src/bot.ml

```diff
@@ -167,93 +166,97 @@ let callback _conn req body =
-        | Some script ->
-            (fun () ->
-              init_git_bare_repository ~bot_info
-              >>= fun () ->
-              action_as_github_app ~bot_info ~key ~app_id
-                ~owner:issue_info.issue.owner ~repo:issue_info.issue.repo
-                (run_coq_minimizer ~script ~comment_thread_id:issue_info.id
-                   ~comment_author:issue_info.user ~owner:issue_info.issue.owner
-                   ~repo:issue_info.issue.repo))
-            |> Lwt.async ;
-            Server.respond_string ~status:`OK ~body:"Handling minimization." ()
-        | None ->
-            Server.respond_string ~status:`OK
-              ~body:(f "Unhandled new issue: %s" body)
-              () )
-      | Ok (signed, CommentCreated comment_info) -> (
-        match coqbot_minimize_text_of_body body with
-        | Some script ->
-            (fun () ->
-              init_git_bare_repository ~bot_info
-              >>= fun () ->
-              action_as_github_app ~bot_info ~key ~app_id
-                ~owner:comment_info.issue.issue.owner
-                ~repo:comment_info.issue.issue.repo
-                (run_coq_minimizer ~script
-                   ~comment_thread_id:comment_info.issue.id
-                   ~comment_author:comment_info.author
-                   ~owner:comment_info.issue.issue.owner
-                   ~repo:comment_info.issue.issue.repo))
-            |> Lwt.async ;
-            Server.respond_string ~status:`OK ~body:"Handling minimization." ()
-        | None ->
-            if
-              string_match
-                ~regexp:
-                  (f "@%s:? [Cc][Ii][- ][Mm]inimize:?\\([^\n]*\\)" bot_name)
-                body
-            then (
-              let requests =
-                Str.matched_group 1 body
-                |> Str.global_replace (Str.regexp "[ ,]+") " "
-                |> String.split ~on:' '
-                |> List.map ~f:Stdlib.String.trim
-                |> List.filter ~f:(fun r -> not (String.is_empty r))
-              in
+          let body = Helpers.trim_comments body in
+          match coqbot_minimize_text_of_body body with
+          | Some script ->
               (fun () ->
                 init_git_bare_repository ~bot_info
                 >>= fun () ->
                 action_as_github_app ~bot_info ~key ~app_id
-                  ~owner:comment_info.issue.issue.owner
-                  ~repo:comment_info.issue.issue.repo
-                  (ci_minimize ~comment_info ~requests ~comment_on_error:true))
+                  ~owner:issue_info.issue.owner ~repo:issue_info.issue.repo
+                  (run_coq_minimizer ~script ~comment_thread_id:issue_info.id
+                     ~comment_author:issue_info.user
+                     ~owner:issue_info.issue.owner ~repo:issue_info.issue.repo))
               |> Lwt.async ;
+              Server.respond_string ~status:`OK ~body:"Handling minimization."
+                ()
```

Comment:
@Zimmi48 Do you understand why ocamlformat introduced this change?  Did I just forget to run ocamlformat on a previous commit, or does it sometimes make non-local changes like this?

Reply:
Or a bug.

Reply:
I just tried it out. I have ocamlformat 0.15.0. Running `dune build @fmt` on the current `master` succeeds (with no changes). Same when running on the PR.

Reply:
I tried it out with `ocamlformat` 0.18 and (despite other formatting changes), the problem is the same. It might be worth a bug report to the `ocamlformat` people, but I'm not sure whether it's worth the effort.